cluster/my-ovm.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kubevirt.io/v1alpha1
+kind: OfflineVirtualMachine
+metadata:
+  name: my-vm
+  labels:
+    my: label
+spec:
+  running: false # equivalent to replicas 0
+  selector:
+    matchLabels:
+      my: label
+  template:
+    metadata:
+      labels: 
+        my: label
+    spec:
+      domain:
+        resources:
+          requests:
+            memory: 8Mi
+        devices:
+          disks:
+          - name: disk0
+            volumeName: mypcv
+      volumes:
+        - name: mypvc
+          persistentVolumeClaim:
+            claimName: myclaim
```

Comment:
The examples are intended to work after a `make cluster-up`. Could you link it to the cirros pvc?

Reply:
I probably just do  not see it. What cirros pvc are we talking about? I can see in other examples, there is direct mount of cirros disk. Is this what should be here?

Reply:
Oh, I meant the registry disk. Just using a cirros registry disk:

```
  volumes:
    - name: registryvolume
      registryDisk:
        image: kubevirt/cirros-registry-disk-demo:devel
```

should be enough.